stackit/internal/services/iaas/network/resource.go

```diff
@@ -183,12 +183,19 @@ func (r *networkResource) Create(ctx context.Context, req resource.CreateRequest
 	// Create new network
 	var httpResp *http.Response
 	ctxWithHTTPResp := runtime.WithCaptureHTTPResponse(ctx, &httpResp)
-	err = r.client.CreateNetwork(ctxWithHTTPResp, projectId).CreateNetworkPayload(*payload).Execute()
+	network, err := r.client.CreateNetwork(ctxWithHTTPResp, projectId).CreateNetworkPayload(*payload).Execute()
 	if err != nil {
 		core.LogAndAddError(ctx, &resp.Diagnostics, "Error creating network", fmt.Sprintf("Calling API: %v", err))
 		return
 	}
-	network, err := wait.CreateNetworkWaitHandler(ctx, r.client, projectId, httpResp.Header.Get("x-request-id")).WaitWithContext(context.Background())
+
+	tflog.Info(ctx, "[IaaS API] Triggered creation of network.\n", map[string]interface{}{
+		"Network ID":    *network.NetworkId,
+		"Network State": *network.State,
+	})
+	tflog.Info(ctx, "Waiting for network to be created...")
```

Comment:
I don't think we need this logging here, unless I'm missing something, I don't see it anywhere else. Also because Terraform has built-in those "creating (...) ..." so we don't need to say it ourselves

In this case, I think we can just ignore the output of `CreateNetwork` with a `_` because we will wait anyway and will get the complete network there